### PR TITLE
Jul 26, 2023

### PR DESCRIPTION
**Pull request by nickdesaulniers**
Jul 26, 2023

### CODE REVIEW

**Comment by nickdesaulniers** [comment]
After lunch I'll add a link to https://github.com/ClangBuiltLinux/reduce/pull/1.  I had the notes in github's UI, but I lost them when I misclicked, so these were hastily rematerialized.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ invite and meet link.
 
 Send to: llvm@lists.linux.dev
 
+## Jul 26, 2023
+- [asm goto error diagnostic fixed](https://github.com/ClangBuiltLinux/linux/issues/1886)
+- [improving constant expression evaluation](https://github.com/ClangBuiltLinux/linux/issues/1889)
+- [perf LTO](https://lore.kernel.org/llvm/20230724201247.748146-1-irogers@google.com/)
+- LoongArch LLD support landed in clang-17
+  - tuxmake work for CI; hexagon prior-art?
```
Comment (nathanchance) [inline README.md:14]: https://github.com/ClangBuiltLinux/continuous-integration2/issues/599

File: README.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ invite and meet link.
...
+- [qemu x86 tcg issue](https://github.com/ClangBuiltLinux/boot-utils/issues/111)
+- [powerpc missing instruction](https://github.com/ClangBuiltLinux/linux/issues/1891)
+- Discussion around promoting W=1 warnings to always on, even if some architectures have lingering instances
+- release/17.x branched
```
Comment (nathanchance) [inline README.md:18]: For CI: https://github.com/ClangBuiltLinux/continuous-integration2/issues/608

File: README.md [hunk 3]
```diff
@@ -6,6 +6,19 @@ invite and meet link.
...
+- [`-fbounds-safety` video published](https://youtu.be/RK9bfrsMdAM)
+- [KASAN + LTO fixes](https://github.com/ClangBuiltLinux/linux/issues/1721)
+
```
Comment (nathanchance) [inline README.md:21]: Consider including https://github.com/ClangBuiltLinux/linux/issues/1890 as well.

**Review by nathanchance — APPROVED** [review]
LGTM, I added some links that may be relevant.

File: README.md (hunk 1 above)
Comment (nickdesaulniers) [reply README.md]: done PTAL

File: README.md (hunk 3 above)
Comment (nickdesaulniers) [reply README.md:24]: done PTAL

File: README.md (hunk 2 above)
Comment (nickdesaulniers) [reply README.md:18]: done PTAL

**Pull request merged**